src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -68,9 +68,9 @@ type ConfirmProps = { accountId: AccountId | undefined } & StepComponentProps
 
 // Estimated miner fees are approximative since there might be a reconciliation Tx
 // and an actual savers Tx with different fees, and we're doubling the fees
-// So the actual buffer is at best 3 Txs (4 -1 Tx), at worst 2 Txs (4 - 2 Txs)
-// Or even a bit less
-const TXS_BUFFER = 8
+// This does NOT ensure dust will be kept for future Txs, but will ensure we are conservative WRT gas used
+// so that the final outbound Tx can go through
```

Comment:
At best this saves dust, but given the ramifications of the pre-deposit Tx, this mostly serves as a safety belt for the Tx to go through